frontend-react/src/components/annotate.js

```diff
@@ -29,15 +28,20 @@ class Annotate extends Component {
         if(uploaded){
           return (
               <div className="border">
-      
+                  <form className="selectForm" onSubmit={this._handleSubmit}>
+                    <input type="file" id="file" multiple={true} accept="image/*" className="fileInputButton" onChange={this.fileSelectedHandler} />
+                    <label htmlFor="file" className="fileLabel">
+                      Add Images
+                    </label>
+                  </form>
                 <div className="imageDisplay">
                   <ReactImageAnnotate
-                    taskDescription="# Draw region around each Kiwifruit bunch"
-                    images= {data} 
-                    regionClsList={["kiwi fruit", "leaf"]}
```

Comment:
I am no longer able to add a label to annotations. I think we may need to leave some default annotations or the empty string `regionClsList={[""]}` inside of here.